wandb/__init__.py

```diff
@@ -284,7 +285,34 @@ def load_ipython_extension(ipython):
     pass
 
 
-def jupyter_login(force=True, api=None):
+def login(anonymous="never", key=None):
+    """Ensure this machine is logged in
+
+       You can manually specify a key, but this method is intended to prompt for user input.
+
+       anonymous can be "never", "must", or "allow".  If set to "must" we'll always login anonymously,
+       if set to "allow" we'll only create an anonymous user if the user isn't already logged in.
+    """
+    # This ensures we have a global api object
+    ensure_configured()
+    os.environ[env.ALLOW_ANONYMOUS] = anonymous
```

Comment:
Can you change `env.ALLOW_ANONYMOUS` to `env.ANONYMOUS` (and the underlying env variable to `WANDB_ANONYMOUS`.